codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ValueEntitySourceGeneratorSuite.scala

```diff
@@ -77,4 +77,36 @@ class ValueEntitySourceGeneratorSuite extends munit.FunSuite {
           |""".stripMargin)
   }
 
+  test("it can generate a provider") {
+    val file =
+      provider(testData.valueEntity(domainParent), testData.simpleEntityService(apiParent))
+    assertNoDiff(
+      file.content,
```

Comment:
ok, then we have a cleanup task to align all other generation to return File instead of String
(consistency police here)

Reply:
I'd like that ;) (and yeah, we have a bunch of consistency things to clean up ;) ). Will PR.